.github/workflows/pypi_build.yaml

```diff
@@ -0,0 +1,94 @@
+name: Create and publish pypi image
+
+on:
+  push:
+  # only publish with a new tag
+    # branches: 
+    #   - develop
+    #   - main
```

Comment:
As a general rule of thumb, commented out code should be removed when merging. If you need the code later on, it can be retrieved from the git commit history

Reply:
ok! Thanks :)